lib/complete-provider.coffee

```diff
@@ -36,7 +35,13 @@ PREFIX_IMPORT_FUNCTIONS    = new RegExp("[\\(\\s,]+(" + REGEXP_FUNCTION_NAME_MAY
 # regexps for completion in functions
 PREFIX_FUNCTION_NAME       = new RegExp("(" + REGEXP_FUNCTION_NAME_JUST + "|" + REGEXP_ALIAS_NAME_CAP + "\\." + REGEXP_FUNCTION_NAME_MAYBE + ")$")
 
-class CompleteProvider extends Provider
+class Suggestion
+  constructor: (@provider, options) ->
```

Comment:
You do not need to track @provider. Consider removing it, it's not used by `autocomplete-plus`.